addons/tutorial_01_get_started/manifest.json

```diff
@@ -20,8 +20,8 @@
           "op": "vpn_off"
         },{
           "op": "vpn_location_set",
-          "exitCountryCode": "at",
-          "exitCity": "Vienna",
```

Comment:
Why was this changed?

Reply:
because Austria is the second country in the list (at least in most of the languages). And if Australia is not usable, we switch to the second one, which must be:
- not opened
- not already selected

Reply:
would it be feasible to add a check to the `hasAvailableCities` property to make sure it has at least one unselected city, and then use that to query for the first country with `hasAvailableCities=true`?

Reply:
Yes, but we do not support scrolling in tutorial yet...